lib/helpers/config-store.js

```diff
@@ -27,6 +27,31 @@ function readJSONFromDisk(path) {
 }
 
 export default {
+    /**
+     * Load the configuration from an exact path and return an EikConfig object
+     *
+     * @param {string} configFilePathname
+     * @param {function} loadJSONFromDisk
+     *
+     * @returns {EikConfig}
+     */
+    loadExact(configFilePathname, loadJSONFromDisk = readJSONFromDisk) {
```

Comment:
```suggestion
    loadFromPath(configFilePathname, loadJSONFromDisk = readJSONFromDisk) {
```
I would rename this to what it actually does.
One thing I was thinking reading this, was that the loader function could've been the only argument too. It does look a bit strange to have both the path and the loader function.
This is what is done in the test of `config-store.test.js`, where you pass in the path even we are just returning some mock data.